site/app/templates/submission/grade_inquiry/Discussion.twig

```diff
@@ -125,6 +125,9 @@
     </div>
 
     <script>
+        window.onload = function() {
+            loadDraft();
+        };      
```

Comment:
Can this be moved to an external JS file?  Also, it would be better to use jQuery for this: `$(document).ready(() => {...`